README.md

```diff
@@ -40,14 +40,14 @@ dispatch:
     search_order: ['spark_utils', 'dbt_utils']
 ```
 
-## Step 2: Install the package (skip if also using the `linkedin` transformation package)
+## Step 2: Install the package (skip if also using the `linkedin` transformation package, or `ad_reporting` combination package)
```

Comment:
can you add the Ad Reporting link to line 44 as well?

Reply:
Added.